### PR TITLE
DEV-3995-make-the-blockchain-branch-audits-beautiful

### PR DESCRIPTION
**Pull request by mikysett**

### CODE REVIEW

File: subjects/blockchain/benchmarking-tool/audit/README.md [hunk 1]
```diff
@@ -1,37 +0,0 @@
-## Benchmarking tool audit
```
Comment (nprimo) [inline subjects/blockchain/benchmarking-tool/audit/README.md:1]: Was this deletion on purpose?

File: subjects/blockchain/financial-instruments/audit/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+## Financial Instruments Audit
```
Comment (nprimo) [inline subjects/blockchain/financial-instruments/audit/README.md:1]: This should be `h4`

File: subjects/blockchain/nft-marketplace/audit/README.md [hunk 3]
```diff
@@ -1,59 +1,59 @@
-#### NFT Marketplace
```
Comment (nprimo) [inline subjects/blockchain/nft-marketplace/audit/README.md:1]: This should be an `h4`

File: subjects/blockchain/financial-instruments/audit/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+##### As the second user, Beatriz, open the CLV page and buys 10 CLV at market price
+###### Does the platform correctly offer to buy at 9 TRG?
+###### Does the trade seem to execute?
+###### Does Beatriz have 60 TRG and 10 CLV now?
+###### Does Beatriz portfolio display a withdrawal button for CLV?
+
+##### From the portfolio page, click on withdraw for CLV
+###### Does the transaction execute?
+###### Does Beatriz now have full possession of its assets (in the second column)?
```
Comment (nprimo) [inline subjects/blockchain/financial-instruments/audit/README.md:35]: Usually there is a line spacing between all the lines in the audits - it seems to not affect the final rendering, but it would be nice to keep the style consistent in my opinion

File: subjects/blockchain/node-dashboard/audit/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+## Node Dashboard audit
```
Comment (nprimo) [inline subjects/blockchain/node-dashboard/audit/README.md:1]: This should be `h4`

File: subjects/blockchain/node-dashboard/audit/README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+###### Does the interface display the latest block of each node?
+###### Does the interface display their Ethereum address and balance?
+###### Does the interface display the CPU and memory consumption of each node?
+
+##### Launch scenario 0
+###### Does Alice, Bob and Cassandra have a positive balance of ETH?
+
+##### Launch scenario 1
+###### Does the command provide feedback?
+###### Does the `infos` command show the updated ETH balance?
```
Comment (nprimo) [inline subjects/blockchain/node-dashboard/audit/README.md:19]: the code rendering for the `h6` in the audit is a bit off: it always renders it as a code block (new line with just this word). I would use bold or italics to highlight the keyword instead

File: subjects/blockchain/node-dashboard/audit/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+###### Does the interface display their Ethereum address and balance?
+###### Does the interface display the CPU and memory consumption of each node?
+
+##### Launch scenario 0
+###### Does Alice, Bob and Cassandra have a positive balance of ETH?
+
+##### Launch scenario 1
+###### Does the command provide feedback?
+###### Does the `infos` command show the updated ETH balance? 
+###### Does the `infos` command displays transactions in the mempool?
```
Comment (nprimo) [inline subjects/blockchain/node-dashboard/audit/README.md:20]: same as above

File: subjects/blockchain/node-dashboard/audit/README.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+##### Launch scenario 2
+###### Does the command provide feedback?
+###### Do Driss and Elena appear to have received 1000 BY tokens?
+
+##### Launch scenario 3
+###### Does the command provide feedback?
+###### Does the `infos` command show the updated balance of Elena receiving one ETH?
```
Comment (nprimo) [inline subjects/blockchain/node-dashboard/audit/README.md:28]: Same as above

File: subjects/blockchain/node-dashboard/audit/README.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+##### Run `temporary-failure Alice` and launch `infos`
+###### Is the node disabled in the `infos` output?
```
Comment (nprimo) [inline subjects/blockchain/node-dashboard/audit/README.md:31]: Same as above

File: subjects/blockchain/node-dashboard/audit/README.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+###### Does it come back online after 40 seconds?
+###### Does it appear to be up to date a couple of minutes later?
+
+#### Bonus 
+###### +Does the `-u` option allow you to launch each command with a regular update?
```
Comment (nprimo) [inline subjects/blockchain/node-dashboard/audit/README.md:36]: Same as above

File: subjects/blockchain/node-dashboard/audit/README.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+###### +Does the tool offer the possibility of launching a node connected to one of the testnet networks?
```
Comment (nprimo) [inline subjects/blockchain/node-dashboard/audit/README.md:37]: Usually there is a line spacing between all the lines in the audits - it seems to not affect the final rendering, but it would be nice to keep the style consistent in my opinion

File: subjects/blockchain/non-fungible-cats/audit/README.md [hunk 12]
```diff
@@ -14,11 +14,11 @@
...
 
-###### The buy function functions properly
+###### The buy function works properly
 
 ##### transfer cat 1 to a secondary user
 
-###### The transfer function properly works
+###### The transfer function works properly
 
 ##### buy cat 2 at the listed price (10wei)
```
Comment (nprimo) [inline subjects/blockchain/non-fungible-cats/audit/README.md:23]: It seems like a question, such as `Does the transfer work properly?`,  is missing after this action

File: subjects/blockchain/non-fungible-cats/audit/README.md [hunk 13]
```diff
@@ -14,11 +14,11 @@
 
 ##### buy cat 1 at the listed price (10wei)
 
-###### The buy function functions properly
+###### The buy function works properly
```
Comment (nprimo) [inline subjects/blockchain/non-fungible-cats/audit/README.md:17]: It might be better to rephrase this as a yes/no question

File: subjects/blockchain/non-fungible-cats/audit/README.md [hunk 14]
```diff
@@ -14,11 +14,11 @@
...
 
 ##### transfer cat 1 to a secondary user
 
-###### The transfer function properly works
+###### The transfer function works properly
```
Comment (nprimo) [inline subjects/blockchain/non-fungible-cats/audit/README.md:21]: It might be better to rephrase this as a yes/no question. For example

Does the transfer function work properly?

Comment (davhojt) [reply subjects/blockchain/non-fungible-cats/audit/README.md:21]: All must be phrased as questions, with "yes" as the happy path.

File: subjects/blockchain/non-fungible-cats/audit/README.md [hunk 15]
```diff
@@ -48,7 +48,7 @@
 
 ##### Buy cat 0 using the buy button
 
-###### Does the buy function work?
+###### Does the buy function works?
```
Comment (nprimo) [inline subjects/blockchain/non-fungible-cats/audit/README.md:51]: In line 59 there is a missing question mark

File: subjects/blockchain/payment-channel/audit/README.md [hunk 16]
```diff
@@ -1,71 +1,71 @@
-#### Payment Channel
```
Comment (nprimo) [inline subjects/blockchain/payment-channel/audit/README.md:1]: this should be an `h4`

File: subjects/blockchain/sign-service/audit/README.md [hunk 17]
```diff
@@ -1,35 +1,35 @@
-#### Sign Service
```
Comment (nprimo) [inline subjects/blockchain/sign-service/audit/README.md:1]: this should be an `h4`

File: subjects/blockchain/tracking-network/audit/README.md [hunk 18]
```diff
@@ -1,4 +1,4 @@
-#### Tracking Network
+## Tracking Network
```
Comment (nprimo) [inline subjects/blockchain/tracking-network/audit/README.md:1]: this should be an `h4`

**Review by nprimo — CHANGES_REQUESTED** [review]
I left some comments inline for some small things that I think should be changed.

File: subjects/blockchain/non-fungible-cats/audit/README.md (hunk 12 above)
Comment (mikysett) [reply subjects/blockchain/non-fungible-cats/audit/README.md:23]: done

File: subjects/blockchain/node-dashboard/audit/README.md (hunk 11 above)
Comment (mikysett) [reply subjects/blockchain/node-dashboard/audit/README.md:37]: done

File: subjects/blockchain/node-dashboard/audit/README.md (hunk 5 above)
Comment (mikysett) [reply subjects/blockchain/node-dashboard/audit/README.md:1]: done

File: subjects/blockchain/financial-instruments/audit/README.md (hunk 4 above)
Comment (mikysett) [reply subjects/blockchain/financial-instruments/audit/README.md:35]: done

File: subjects/blockchain/nft-marketplace/audit/README.md (hunk 3 above)
Comment (mikysett) [reply subjects/blockchain/nft-marketplace/audit/README.md:1]: done

File: subjects/blockchain/financial-instruments/audit/README.md (hunk 2 above)
Comment (mikysett) [reply subjects/blockchain/financial-instruments/audit/README.md:1]: done

File: subjects/blockchain/non-fungible-cats/audit/README.md (hunk 13 above)
Comment (mikysett) [reply subjects/blockchain/non-fungible-cats/audit/README.md:17]: done

File: subjects/blockchain/non-fungible-cats/audit/README.md (hunk 14 above)
Comment (mikysett) [reply subjects/blockchain/non-fungible-cats/audit/README.md:21]: done

File: subjects/blockchain/non-fungible-cats/audit/README.md (hunk 15 above)
Comment (mikysett) [reply subjects/blockchain/non-fungible-cats/audit/README.md:51]: done

File: subjects/blockchain/payment-channel/audit/README.md (hunk 16 above)
Comment (mikysett) [reply subjects/blockchain/payment-channel/audit/README.md:1]: done

File: subjects/blockchain/sign-service/audit/README.md (hunk 17 above)
Comment (mikysett) [reply subjects/blockchain/sign-service/audit/README.md:1]: done

File: subjects/blockchain/tracking-network/audit/README.md (hunk 18 above)
Comment (mikysett) [reply subjects/blockchain/tracking-network/audit/README.md:1]: done

**Review by nprimo — APPROVED** [review]
LGTM!
